sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -644,7 +696,9 @@ private boolean isInfrastructureClass(Class<?> clazz) {
 			|| clazz == DistrictReferenceDto.class
 			|| clazz == CommunityReferenceDto.class
 			|| clazz == FacilityReferenceDto.class
-			|| clazz == PointOfEntryReferenceDto.class;
+			|| clazz == PointOfEntryReferenceDto.class
+			|| clazz == SubcontinentDto.class
+			|| clazz == ContinentDto.class;
```

Comment:
Can you please add country here as well? Seems like it was forgotten when the entity was introduced

Reply:
The method is used in two lines, none of which seems to make sense:
```java
if (EntityDto.class.isAssignableFrom(field.getType()) && !isInfrastructureClass(field.getType())) {...
```
and
```java
} else if (PersonReferenceDto.class.isAssignableFrom(field.getType()) && !isInfrastructureClass(field.getType())) {
```
Any idea? Whatever happens here, `isInfrastructureClass` is not the deciding factor. If everything works, the method is obsolete.